rpc/transport/ws/server.go

```diff
@@ -71,18 +72,60 @@ func (wsc *serverWebSocketTransport) Close() {
 }
 
 func (wsc *serverWebSocketTransport) Url() string {
-	return "ws://" + webscocketServerAddress + wsc.port
+	return webscocketServerAddress + wsc.port
+}
+
+func (wsc *serverWebSocketTransport) request(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "POST" {
+		http.Error(w, http.StatusText(http.StatusMethodNotAllowed), http.StatusMethodNotAllowed)
+		return
+	}
+	body := http.MaxBytesReader(w, r.Body, 8192)
```

Comment:
Is there any reason we chose `8192` as the maximum amount of bytes we accept for a request? Maybe we could throw it in a constant?

Reply:
This seemed like a reasonable choice. It has been made a constant in c4284943.